app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Contestants</h1>
+<% @contestants.each do |contestant| %>
+    <p><b><%=contestant.name%></b></p>
+    <p>Projects:  </p>
+    <div id="contestant_projects-<%=contestant.id%>">
+      <% contestant.projects.each do |project|%>
+        <ul>
+          <li><%= project.name %></li>
+        </ul>
+     <% end %>
```

Comment:
Since the US asked for the projects to be printed in a comma-separated list all you would have needed to do was use a map with a join at the end like so:

```erb
<p>Projects:
  <span id="contestant_projects-<%=contestant.id%>">
    <%= contestant.projects.map do |project| %>
      <% project.name %>
    <% end.join(", ") %>
  </span>
</p>
```

This way we are implementing the US exactly.